.github/workflows/test.yml

```diff
@@ -154,6 +154,16 @@ jobs:
           tags: gnosispm/safe-config-service:${{ env.DOCKER_TAG }}
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache-new
+      - name: Build and push latest tag
+        id: docker_build_latest
+        if: github.event_name == 'release' && github.event.action == 'released'
+        uses: docker/build-push-action@v3.2.0
+        with:
+          context: .
+          push: true
+          tags: gnosispm/safe-config-service:latest
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache-new
```

Comment:
Maybe it makes more sense to split the jobs (release vs staging).

Before we could just set the `DOCKER_TAG` but now the behaviour is actually different:
- For releases we push two tags (`latest` and the release)
- For commits we push one tag (`staging`)

Reply:
Done @fmrsabino !